app/models/collection_draft.rb

```diff
@@ -1,4 +1,8 @@
 class CollectionDraft < Draft
+  # TODO: we currently allow one to be created, but may allow more in the future
+  # should we just make this has_one for now?
```

Comment:
I don't know about other changes to the table for future functionality or logging. But I don't think there are any table changes necessary to changing the association, I think the changes are only necessary in the models. I think we would need to change:
* obviously the association type from `has_many` to `has_one`
* singularize the association name (to `keyword_recommendation`)
* singularize the association methods (same as above), and I believe we would need to expect the object rather than an ActiveRecord collection/array
* of course change all this back if/when we are going to present recommendations multiple times for a given record

If we were certain that we will be presenting recommendations multiple times in the future, I would not even have added the comment/question. But it seems unclear to me if we will or not and I don't know which is the more likely outcome.